docs/advanced/addons.md

```diff
@@ -24,9 +24,32 @@ v1.1.0 currently ships with two Addons:
 
 _Available as of v1.2.0_
 
-v1.2.0 ships with two more Addons:
+v1.2.0 ships with three more Addons:
```

Comment:
ditto, consider merging with the above and change the text to:
`**v1.2.0 ships with five Addons**`